grails-app/domain/com/unifina/domain/marketplace/Product.groovy

```diff
@@ -100,7 +101,7 @@ class Product {
 			pricePerSecond: pricePerSecond,
 			priceCurrency: priceCurrency.toString(),
 			minimumSubscriptionInSeconds: minimumSubscriptionInSeconds,
-			owner: owner
+			owner: owner.name
```

Comment:
This will lead to N+1 db requests when listing products. I think @harbu had a way to make GORM eager-fetch `owner`. It would be a major improvement in terms of performance.

Reply:
Added optimization!